### PR TITLE
Issue #766: remove sevntu-checkstyle-maven-plugin

### PR DESCRIPTION
**Pull request by rdiachenko**
Issue #766 

`sevntu-checkstyle-maven-plugin` removed

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -28,4 +27,4 @@ Hidoyatov Victor, Troshin Sergey, Svinukhov Vladimir, "Ilia Dubinin":https://git
 "Aleksey Grigirov":https://github.com/KTannenberg, Alexander Berezovsky, "Sergey Burtsev":https://github.com/burtsevsergey, "Baratali Izmailov":https://github.com/baratali, "Max Vetrenko":https://github.com/maxvetrenko,
 "Pavel Baranchikov":https://github.com/pbaranchikov , "Ashutosh Agarwal":https://github.com/Radsaggi, "Alexey Nesterenko":https://github.com/alexkravin. , ......  for whole list please look at "contributors list":https://github.com/sevntu-checkstyle/sevntu.checkstyle/network/members .
 
-The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
\ No newline at end of file
+The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
```
Comment (rnveach) [inline README.textile]: I am not seeing what the change is here.

**Comment by rnveach** [comment]
wercker failure:
> Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Failure to find com.github.sevntu-checkstyle:sevntu-checkstyle-maven-plugin:jar:1.35.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced

**Comment by rnveach** [comment]
travis
> Unexpected argument: maven-plugin

File: README.textile (hunk 1 above)
Comment (rdiachenko) [reply README.textile]: There were no new line symbol at the end of file:
```
git diff README.textile
diff --git a/README.textile b/README.textile
index 6efab66b..aabae016 100644
--- a/README.textile
+++ b/README.textile
@@ -2,7 +2,6 @@ h3. Information "!https://secure.travis-ci.org/sevntu-checkstyle/sevntu.checksty
 
 sevntu-checks: !https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checks.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checks%22
 sevntu-checkstyle-idea-extension:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-idea-extension.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-idea-extension%22
-sevntu-checkstyle-maven-plugin:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-maven-plugin.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-maven-plugin%22
 sevntu-checkstyle-sonar-plugin:!https://img.shields.io/maven-central/v/com.github.sevntu-checkstyle/sevntu-checkstyle-sonar-plugin.svg?label=Maven%20Central!:https://search.maven.org/search?q=g:%22com.github.sevntu-checkstyle%22%20AND%20a:%22sevntu-checkstyle-sonar-plugin%22
 eclipsecs-sevntu-plugin: "all versions":https://github.com/sevntu-checkstyle/sevntu.checkstyle/tree/gh-pages/update-site/plugins
 
@@ -28,4 +27,4 @@ Hidoyatov Victor, Troshin Sergey, Svinukhov Vladimir, "Ilia Dubinin":https://git
 "Aleksey Grigirov":https://github.com/KTannenberg, Alexander Berezovsky, "Sergey Burtsev":https://github.com/burtsevsergey, "Baratali Izmailov":https://github.com/baratali, "Max Vetrenko":https://github.com/maxvetrenko,
 "Pavel Baranchikov":https://github.com/pbaranchikov , "Ashutosh Agarwal":https://github.com/Radsaggi, "Alexey Nesterenko":https://github.com/alexkravin. , ......  for whole list please look at "contributors list":https://github.com/sevntu-checkstyle/sevntu.checkstyle/network/members .
 
-The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
\ No newline at end of file
+The "Jsr305AnnotationsCheck" is based on the "JSR305CheckstylePlugin":https://github.com/bjrke/JSR305CheckstylePlugin by "Jan Burkhardt":https://github.com/bjrke with contributions from Mattias Nissler, Thorsten Ehlers, Fabian Loewner and Ole Langbehn.
```

**Comment by rdiachenko** [comment]
> travis
> 
> > Unexpected argument: maven-plugin

fixed

**Comment by rdiachenko** [comment]
> wercker failure:
> 
> > Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Failure to find com.github.sevntu-checkstyle:sevntu-checkstyle-maven-plugin:jar:1.35.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced

should be fixed after https://github.com/checkstyle/contribution/pull/395 is merged

**Comment by romani** [comment]
@rdiachenko , CI is green. Ok to merge.
Thanks a lot for fix.

In next PR please send update to gh-pages branch ( html site).
We can keep maven artifacts at this branch.

**Comment by rdiachenko** [comment]
> @rdiachenko , CI is green. Ok to merge.
> Thanks a lot for fix.
> 
> In next PR please send update to gh-pages branch ( html site).
> We can keep maven artifacts at this branch.

Done in https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/768. There is also another occurrence in `gsoc-2013-ideas.html` which I didn't remove as it is a sort of historical file.

**Pull request merged**